README.md

```diff
@@ -1,3 +1,329 @@
+library from C++ in a cross-platform manner. As of this writing the bindings
+are functional with GCC on Linux and Mac OS X systems, as well as with Visual
+C++ 2005 and Visual C++ 2008 on Windows.
+
+**Note that the JACE C++ bindings require Java 7 to build and run. They do
+not work with Java 8.**
+
+###Compile-time dependencies
+
+To build the Bio-Formats C++ bindings from source, the following modules are
+required:
+
+- [Apache Maven](http://maven.apache.org/)
+     
+     Maven is a software project management and comprehension tool. Along with
+     Ant, it is one of the supported build systems for the Bio-Formats Java
+     library, and is used to generate the Bio-Formats C++ bindings.
+
+- [CMake](http://www.cmake.org/)
+     
+     CMake is a cross-platform, open source build system generator, commonly
+     used to build C++ projects in a platform-independent manner. CMake
+     supports GNU make as well as Microsoft Visual Studio, allowing the
+     Bio-Formats C++ bindings to be compiled on Windows, Mac OS X, Linux and
+     potentially other platforms.
+
+- [Boost Thread](http://www.boost.org/)
+     
+     Boost is a project providing open source portable C++ source libraries.
+     It has become a suite of de facto standard libraries for C++. The
+     Bio-Formats C++ bindings require the Boost Thread module in order to
+     handle C++ threads in a platform independent way.
+
+- [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/)
+     
+     Version 6 or 7 is required; version 8 is not currently supported.
+     At runtime, only the Java Runtime Environment (JRE) is necessary
+     to execute the Bio-Formats code. However, the full J2SE
+     development kit is required at compile time on some platforms
+     (Windows in particular), since it comes bundled with the JVM
+     shared library (jvm.lib) necessary to link with Java.
+
+For information on installing these dependencies, refer to the section for
+your specific platform below.
+
+###How to build
+
+The process of building the Bio-Formats C++ bindings is divided into two
+steps:
+
+1. Generate a C++ project consisting of "proxies" which wrap the Java code.
+   This step utilizes the Maven project management tool, specifically a
+   Maven plugin called cppwrap.
+
+2. Compile this generated C++ project. This step utilizes the cross-platform
+   CMake build system.
+
+For details on executing these build steps, refer to the section for your
+specific platform below.
+
+###Build results
+
+If all goes well, the build system will:
+
+1. Generate the Bio-Formats C++ proxy classes
+2. Build the Jace C++ library
+3. Build the Java Tools C++ library
+4. Build the Bio-Formats C++ shared library
+5. Build the showinf and minimum_writer command line tools for testing the
+   functionality.
+
+Please be patient, as the build may require several minutes to complete.
+
+Afterwards, the dist/formats-bsd subdirectory will contain the following
```

Comment:
Does jar/ indicate that the BF package jar is contained in this still? What about jace-runtime.jar? This is 2 items down on the existing list